src/Components/NotFoundErrorPage.tsx

```diff
@@ -42,13 +43,23 @@ const NotFound = connect(function NotFound() {
     }
   }, [])
 
+  useEffect(() => {
+    redirectIfNeeded()
+
+    async function redirectIfNeeded() {
```

Comment:
* [ ] FYI the naming could tell more about the purpose of the effect, e.g., `beginEditingIfContentIsEmpty()`.